tools/run-mypy

```diff
@@ -16,6 +16,7 @@ TOOLS_DIR = os.path.dirname(os.path.abspath(__file__))
 os.chdir(os.path.dirname(TOOLS_DIR))
 
 exclude_common = """
+zerver/tests/webhooks/test_mailchimp.py
```

Comment:
Why was this file added to the excludes?
We should be adding mypy annotations to any new files.